src/v2i-hub/CARMACloudPlugin/manifest.json

```diff
@@ -28,6 +28,21 @@
 			"default":"127.0.0.1",
 			"description":"Server IP address for V2X hub"
 		},
+		{
+			"key":"WebServicePort",
```

Comment:
This name is a bit generic if this port is infact only for TCMs. Maybe rename it something more specific to reflect it's usage?

Reply:
I can rename it but this is not the newly created configuration parameter for carma cloud port and ip. It is the existing one for the IP and port to deploy the CARMA-CLoud Plugin REST Server. Do you have any suggestions for better naming convention. V2X-Hub CARMA Cloud REST Server Port?

Reply:
Ah, if this is pre-existing I'm fine to leave it for now. But something like that, would be preferable. "Web Service Port" just leads me to ask "What web service"? and will likely get slightly more confusing when the new UI framework stuff is finalized, as there may be multiple web services running.